.env

```diff
@@ -78,6 +78,13 @@ REACT_APP_SECOND_EXTRA_PROP_SHOW_IN_RESULTS=
 # leave blank to keep this property a free form text field
 REACT_APP_SECOND_EXTRA_PROP_DROPDOWN_LABELS=
 
+######################################################################################
+# PV Details Page
+######################################################################################
+
+# Set to true to start live monitoring when visiting a PV details page
```

Comment:
Could you update the comment something like this:

# Only used if PVWS is enabled
# Set to true ...
# If false, web socket will be opened briefly on page load to display "static" PV meta data